src/App.css

```diff
@@ -178,8 +181,51 @@ nav {
   text-decoration: none;
 }
 
+.btn {
+  font-family: var(--font-family-body-text);
+  background-color: white;
+  color: var(--primary-color);
+  font-size: 1.3rem;
+  gap: 1.5rem;
+  line-height: 2;
+  height: var(--btn-height);
+  transition: all 200ms linear;
+  border-radius: 4px;
+  width: var(--btn-icon-width);
```

Comment:
Wouldn't this mean the buttons are square? How do you accommodate for rectangular buttons?

Reply:
There's a different class for the non-square buttons below 👇

Reply:
@segdeha : I just added two CSS class for that ^^: `.btn__icon` & `.btn__link`. I hope it's much better.